app/models/ckb_sync/new_node_data_processor.rb

```diff
@@ -282,13 +282,13 @@ def process_withdraw_dao_events!(local_block, new_dao_depositors, dao_contract)
     def process_interest_dao_events!(local_block, dao_contract)
       addrs_withdraw_info = {}
       claimed_compensation = 0
-      local_block.cell_inputs.nervos_dao_withdrawing.select(:id, :ckb_transaction_id,
```

Comment:
No use for block_id?

Reply:
The `dao`  field in `cell_outputs` is redudent to the `blocks.dao`.
I will remove the `cell_outputs.dao` in the future.
And save the parsed dao headers in the `blocks` table.